programs/ssl/ssl_client2.c

```diff
@@ -602,7 +661,8 @@ int main( int argc, char *argv[] )
         else if( strcmp( p, "request_size" ) == 0 )
         {
             opt.request_size = atoi( q );
-            if( opt.request_size < 0 || opt.request_size > MBEDTLS_SSL_MAX_CONTENT_LEN )
+            if( opt.request_size < 0 ||
+                opt.request_size > MBEDTLS_SSL_MAX_CONTENT_LEN )
```

Comment:
Could you please revert this whitespace change? There's been a semantic change in `development`, and removing this whitespace change would make the file mergeable.

Reply:
Done in b6f880b.